WebApp/LegendPanel.ascx

```diff
@@ -17,13 +17,13 @@ limitations under the License.
 <%@ Control Language="C#" AutoEventWireup="true" CodeFile="LegendPanel.ascx.cs" Inherits="LegendPanel" %>
 <%@ Register TagPrefix="gpv" Assembly="App_Code" Namespace="GPV" %>
 
-<div class="topHead">
+<div class="topHead FunctionHeader">
```

Comment:
What is ```FunctionHeader``` and why it moved to the above element.

Reply:
FunctionHeader used for changing header color and background color.